contracts/EasyAuction.sol

```diff
@@ -176,7 +182,17 @@ contract EasyAuction is Ownable {
             minFundingThreshold,
             false
         );
-        emit NewAuction(auctionCounter, _auctioningToken, _biddingToken);
+        emit NewAuction(
+            auctionCounter,
+            _auctioningToken,
+            _biddingToken,
+            block.timestamp + orderCancelationPeriodDuration,
+            block.timestamp + duration,
```

Comment:
If we have space in the stack we could make these two sums into variables, in this way we don't need to remember to change both if we want to change how the durations are computed. Alternatively, we could add a test.